deepspeed/utils/mappings.py

```diff
@@ -0,0 +1,95 @@
+'''
```

Comment:
If this is only used for MoE can we move it to `deepspeed/moe`?

Reply:
Yes, it's only used for MoEs. I'll move this file.